float64[6] pos
float32 speed
---
bool status